CONTRIBUTING.md

```diff
@@ -152,6 +152,31 @@ To create a new test:
 Snapshots are created and stored in the `integ.example.ts.snapshot` directory and should be committed along with
 the rest of your code.
 
+## Linux/MacOS Environment Setup
+
+### Installation
+
+Setup Node.js on your environment:
+
+* [Node.js Install](https://nodejs.org/en/download/package-manager)
```

Comment:
Should we be encouraging nvm usage instead of direct installs?

Reply:
I think that it is out of scope of this library how exactly users install Node